src/CreateItem.js

```diff
@@ -8,11 +9,12 @@ class CreateItem extends Component {
 
     render() {
         return (
-            <div>
+            <div className="CreateList">
                 <form>
-                    <input placeholder="Blurb" />
+                    <input placeholder="Blurb" className=""/>
```

Comment:
Blank className?

Reply:
I will finish the CSS after connecting with the firebase.